backend/protocol_rpc/server.py

```
@@ -191,143 +203,161 @@
+                                    restore_transaction["hash"], False
+                                )
+                                local_transactions_processor.reset_consensus_history(
+                                    restore_transaction["hash"]
+                                )
+                                local_transactions_processor.set_transaction_timestamp_appeal(
+                                    restore_transaction["hash"], None
+                                )
+                                local_transactions_processor.reset_transaction_appeal_processing_time(
+                                    restore_transaction["hash"]
+                                )
+                        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)